config/job_conf.xml.interactivetools

```diff
@@ -21,21 +20,16 @@
                If you develop interactive tools on your 'localhost' and don't have a proper domain name
                you need to tell all Docker containers a hostname where Galaxy is running.
                This can be done via the add-host parameter during the `docker run` command.
-               'my-hostname' here is an arbritrary hostname that matches the IP address of your
-               Galaxy host. Make sure this hostname ('my-hostname') is also set in your galaxy.yml file, e.g.
-               `galaxy_infrastructure_url: http://my-hostname:8080`.
-              Note: XML does not allow double hyphens in comments, hence the line below has them escaped as &#45;
+               'localhost' here is an arbitrary hostname that matches the IP address of your
+               Galaxy host. Make sure this hostname ('localhost') is also set in your galaxy.yml file, e.g.
+               `galaxy_infrastructure_url: http://localhost:8080`.
           -->
-          <!-- <param id="docker_run_extra_arguments">&#45;&#45;add-host my-hostname:10.1.2.3</param> -->
-
+          <param id="docker_run_extra_arguments">--add-host localhost:host-gateway</param>
```

Comment:
"host.docker.internal" maybe to match the example from the galaxy.yml?

Reply:
Which example ? The hostname needs to match `galaxy_infrastructure_url`, which is set to `http://localhost:8080` in galaxy.yml.interactivetools. I feel like `host.docker.internal` gives you the idea that this is some magic address that can't be changed.

Reply:
https://github.com/galaxyproject/galaxy/pull/13470/files#diff-dc06bbcb59cc0db161563c300c1084e592aead949dbdc6f88b4687040344662cR12  this here ..

Reply:
Alright, I missed that. Better in https://github.com/galaxyproject/galaxy/pull/13470/commits/ed8ce47c42f6fe216409907dfec31d0b8811be7b ?